PEP 11: Clarify Windows support phases (#2801)
python/peps: pep-0011.txt

[1]_. Each product's lifecycle has a mainstream support phase, where
the product is generally commercially available, and an extended
support phase, where paid support is still available, and certain bug
fixes are released. Sometimes this is then followed by a third phase

> called "ESU", for "extended security updates".

Reviewer: Perhaps mention ESU is a program of last resort? And link to https://learn.microsoft.com/en-us/lifecycle/faq/extended-security-updates

> fixes are released. Sometimes this is then followed by a third phase
> called "ESU", for "extended security updates".

Reviewer 2: I would (and did, in https://github.com/python/cpython/issues/76773#issuecomment-1093773623 ) describe ESU not as a "phase" but rather a specialized, paid program for high-volume enterprise customers, and be more explicit here (as well as below) in mentioning ESU specifically in the context of making clear the distinction with the Extended Support phase and CPython support, rather than making it part of "this lifecycle" that CPython is stated to follow below.

So, how about (incorporating @hugovk 's comments as well):

```suggestion
fixes are released. This is distinct from `ESU (Extended Security Updates)
<https://learn.microsoft.com/en-us/lifecycle/faq/extended-security-updates>`_,
a specialized, paid program available to high-volume enterprise customers
as a "last resort" option after the end of extended support.
```

CPython's Windows support now follows this lifecycle. A new feature
release X.Y.0 will support all Windows releases whose extended support

> phase is not yet expired. (We don't consider the ESU phase for this
> purpose; only the "extended support" phase.) Subsequent bug fix releases
> will support the same Windows releases as the original feature release
> (even if the extended support phase has ended).

Reviewer 2: ```suggestion
phase is not yet expired. (ESU is not considered supported.)
Subsequent bug fix releases will support the same Windows releases
as the original feature release (even if the extended support phase has ended).
```

Following the above, clarifying that ESU is a specialized program, not a phase, and avoiding repeating ourselves about "extended support" too many times.
